Package.swift

```diff
@@ -0,0 +1,39 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "FirebaseManager",
+    platforms: [
+        .iOS(.v14),
+        .macOS(.v10_15)
+    ],
+    products: [
+        .library(
+            name: "FirebaseManagerPackage",
+            targets: [
+                "FirebaseStorageManager"
+            ])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/firebase/firebase-ios-sdk.git",
+                 .branch("master"))
```

Comment:
Do we want the master here? Maybe the safer option is up to next major level if there are any breaking changes?

Reply:
Yes, changed it!